catalogue_pipeline/recorder/src/main/scala/uk/ac/wellcome/platform/recorder/services/RecorderWorkerService.scala

```diff
@@ -2,23 +2,39 @@ package uk.ac.wellcome.platform.recorder.services
 
 import akka.actor.ActorSystem
 import com.google.inject.Inject
-import io.circe.Decoder
+import com.gu.scanamo.DynamoFormat
+import io.circe.{Decoder, Encoder}
+import uk.ac.wellcome.messaging.sqs.{SQSReader, SQSWorkerToDynamo}
 import uk.ac.wellcome.metrics.MetricsSender
-import uk.ac.wellcome.sqs.{SQSReader, SQSWorkerToDynamo}
+import uk.ac.wellcome.models.SourceMetadata
 import uk.ac.wellcome.models.work.internal.UnidentifiedWork
+import uk.ac.wellcome.recorder.models.RecorderWorkEntry
+import uk.ac.wellcome.storage.VersionedHybridStore
 import uk.ac.wellcome.utils.JsonUtil._
 
 import scala.concurrent.Future
 
+case class EmptyMetadata()
+
 class RecorderWorkerService @Inject()(
+  versionedHybridStore: VersionedHybridStore[RecorderWorkEntry],
   reader: SQSReader,
   system: ActorSystem,
   metrics: MetricsSender
 ) extends SQSWorkerToDynamo[UnidentifiedWork](reader, system, metrics) {
 
-  override implicit val decoder = implicitly[Decoder[UnidentifiedWork]]
+  implicit val decoder = Decoder[RecorderWorkEntry]
+  implicit val encoder = Encoder[RecorderWorkEntry]
+  implicit val dynamoFormat = DynamoFormat[RecorderWorkEntry]
```

Comment:
If this is the line that it's whingeing about, you might need to do `import uk.ac.wellcome.dynamo._`